index.html

```diff
@@ -49,9 +49,7 @@ <h1>I’m Baraka<br>Glad to see you!</h1>
     </section>
     <section id="portfolio">
         <div class="portfolio">
-            <div class="snapshot1">
-                <!-- <img src="./images/Snapshoot Portfolio.png" alt=""> -->
```

Comment:
Everything is very well. It would be better if you remove unused comments from your code. It will increase the compile speed. 😄